virttest/utils_libvirt/libvirt_cpu.py

```diff
@@ -38,9 +39,10 @@ def add_cpu_settings(vmxml, params):
     else:
         cpu_xml = vm_xml.VMCPUXML()
 
-    if cpu_xml.xmltreefile.find('mode'):
+    try:
```

Comment:
Using try-except to control flow is not quite recommended especially when there are other solutions. 
`if cpu.xmltreefile.getroot().get('mode'):`
or
`if 'mode' in cpu.xmltreefile.getroot().attrib:`
could solve the issue.

The reason that  `cpu_xml.xmltreefile.find('mode')` can't find `mode` is because `mode` is an attribute in `cpu` tag, not a nested tag like `model`

Reply:
It's ok for me to use any way that can wok. Will you prefer to submit a PR or change it in this PR?

Reply:
I agree with @chloerh. I think it's better to use `if block` instead of `try-except `which is not good for performance.

Reply:
@yingshun, this exception happened in a very small rate.  Again, it's ok for me to use any one to make the function work well.

Reply:
Changed as @chloerh suggested. And the test result is as below -

avocado run --vt-type libvirt virsh.migrate_mem.mem_device
JOB ID : 348e3136ea4af360f6185745e6ba766e441bed87a
JOB LOG : /root/avocado/job-results/job-2021-03-15T04.15-1bed87a/job.log
(1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_mem.mem_device: PASS (129.36 s)
RESULTS : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0